core/src/main/java/org/apache/accumulo/core/util/Merge.java

```diff
@@ -225,8 +226,9 @@ protected void merge(AccumuloClient client, String table, List<Size> sizes, int
     try {
       Text start = sizes.get(0).extent.prevEndRow();
       Text end = sizes.get(numToMerge - 1).extent.endRow();
-      message("Merging %d tablets from (%s to %s]", numToMerge, start == null ? "-inf" : start,
-          end == null ? "+inf" : end);
+      message("Merging %d tablets from (%s to %s]", numToMerge,
+          start == null ? "-inf" : Key.toPrintableString(start.getBytes(), 0, start.getLength(), end.getLength()),
```

Comment:
I believe the last parameter should be `start.getLength()`.

Reply:
ah, definitely, thanks! Fix in flight.